apps/sms/views/inbox/js/inbox.js

```diff
@@ -124,6 +125,16 @@ var InboxView = {
       groupThreadTitleMaxLength: (window.innerWidth - 100) / 10
     });
 
+    this.once('fully-loaded', () => {
+      this.ensureReadAheadSetting();
+
+      this.readyDeferred.resolve();
```

Comment:
Btw, this ```readyDeferred``` is used only in ```InboxView.whenReady``` that is used only by ConversationView :) And it basically is resolved when ```App.whenReady``` is resolved. so we probably can get rid of this method entirely and switch ConversationView to ```App.whenReady``` instead.